### DISCUSSION TITLE
EXE does not open

### DISCUSSION THREAD

**Discussion by LiquidState317** [num0005/Osoyoos-Launcher#17] [General]
The bug is... the exe wont open

**Comment by num0005** [comment]
Can you provide more details?

**Comment by LiquidState317** [comment]
the only details are that I download it from github just like a youtube
video, but it does not open, the exe just doesnt run

**Comment by num0005** [comment]
What operating system do you use? And did you make sure to install the dotnet runtime?

**Comment by haloizcool** [comment]
So, I'm having the exact same problem as LiquidState317. I've downloaded the correct dotnet runtime as osoyoos was working earlier today. I'm running windows 10. Regardless of whether I run as admin or not osoyoos begins to run in task manager then after a short time fails to open.

**Reply by num0005** [reply]
Could you check the event log?

**Reply by num0005** [reply]
https://www.howtogeek.com/123646/htg-explains-what-the-windows-event-viewer-is-and-how-you-can-use-it/

**Reply by haloizcool** [reply]
![Osoyoos Error 1](https://user-images.githubusercontent.com/96149860/146583379-ef50b00f-968e-4d83-b862-4a7574d20e4a.PNG)
![Osoyoos Error 2](https://user-images.githubusercontent.com/96149860/146583397-714404d3-8233-496e-9631-f9222c0928a5.PNG)

Is this what you need?

**Reply by num0005** [reply]
Yes thank you, it looks like it's crashing while trying to handle an exception.

**Reply by num0005** [reply]
Try the new release

**Comment by General-101** [comment]
@LiquidState317 @haloizcool 
Can either of you contact me on Discord?
General_101#9814

Github is kinda trash for tech support related stuff. If not I guess we can continue this on Github.
